src/v/redpanda/admin_server.h

```diff
@@ -14,6 +14,7 @@
 #include "config/endpoint_tls_config.h"
 #include "coproc/partition_manager.h"
 #include "model/metadata.h"
+#include "redpanda/drain_manager.h"
```

Comment:
Only a forward declaration is required (which true of some of the other includes, too) - this is included from `redpanda/application.h`

Reply:
fixed